xml/System.Security.Cryptography.X509Certificates/X509Certificate2.xml

```diff
@@ -3228,8 +3228,8 @@ The Subject Alternative Name extension or Subject Name could not be decoded.</ex
         <ReturnType>System.Byte[]</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the raw data of a certificate.</summary>
-        <value>The raw data of the certificate as a byte array.</value>
+        <summary>Gets the raw X.509 public data of a certificate.</summary>
+        <value>Gets the raw X.509 public data of a certificate.</value>
```

Comment:
```suggestion
        <value>The raw X.509 public data of a certificate.</value>
```

It's... just how properties work.  `<summary>` says "Gets blah blah" or "Gets or sets blah blah", and `<value>` just says "Blah blah".

Reply:
Here's the guidance for [value](https://github.com/dotnet/dotnet-api-docs/wiki/Property-Value) and [summary](https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Property).